drivers/gpu/drm/vc4/vc4_dpi.c

```diff
@@ -361,6 +369,20 @@ static int vc4_dpi_bind(struct device *dev, struct device *master, void *data)
 	if (ret)
 		return ret;
 
+	dpi->rgb_order_override  = -1;
```

Comment:
There's a stray extra space on this line.

Reply:
Fixed.